### DISCUSSION TITLE
Slot card light

### DISCUSSION THREAD

**Discussion by kompressor** [cedricp/ddt4all#1042] [Q&A]
Hello! After activate (DDT4ALL) some features, and solved a couple of problems, I notice that theres is no light on the slot card, i. e. there is light but just by 1 sec. Can some help?

**Comment by KarelSvo** [comment]
No information about vehicle type, ECU, selected ECU file.................................

**Comment by kompressor** [comment]
Jesus.... This is very complicated for an old lad... The car is Megane from 2014 - Confort SS.
